nix/integration-test.nix

```
@@ -0,0 +1,47 @@
+# Check `todo-app` integration with external services.
+{ self', pkgs, ... }:
+{
+  process-compose.services = { config, ... }: {
```

To avoid the "services.services" option namespace confusion:

```suggestion
  process-compose.todo-app-stack = { config, ... }: {
```

---

Should I create two configs, namely `todo-app-stack-test` and `todo-app-stack`? The latter will not have the `test` process and will be used for local development?

---

I have no opinion on that. Just wanted the name to be descriptive ("todo-stack") rather than generic ("services").

---

But calling it `todo-app-stack` might not be appropriate, because we only run external-services there as `todo-app` is not a server

---

`todo-app-services` then?

---

https://github.com/Platonic-Systems/process-compose-flake/issues/27 would be nice to have because for `todo-app-stack-test`, which I am thinking of calling `todo-app-integ-test`, would only require auto-wiring for `checks` and not `packages`